deploy/06_deploy_coverage_pool.js

```diff
@@ -9,7 +9,10 @@ module.exports = async ({ getNamedAccounts, deployments }) => {
     log: true,
   })
 
-  if ((await read("AssetPool", "owner")) !== CoveragePool.address) {
+  if (
+    ethers.utils.getAddress(await read("AssetPool", "owner")) !==
+    ethers.utils.getAddress(CoveragePool.address)
```

Comment:
Seems like it can be reusable in other scripts, so maybe we can extract to a separate function or even add to the [hre object](https://hardhat.org/advanced/hardhat-runtime-environment.html#extending-the-hre). Ofc non-blocking comment!

```
function isSameAddress(address1, address2) {
 ethers.utils.getAddress(address1) !==
 ethers.utils.getAddress(address2)
}
```

Reply:
Good idea, I'll handle it in a follow-up PR.

Reply:
Covered in https://github.com/keep-network/coverage-pools/pull/162